### PR TITLE
Finish Shop Backend

### PR DESCRIPTION
**Pull request by realCalvin**
- Changed some file names and a little bit of the badge titles/description to not have to deal with char ignoring the single quotes
- Shop backend fully works, deducts points/checks if user has enough, etc
- Has alerts up top that disappears after 3 seconds. (not enough credits or successfully purchased)

image

### CODE REVIEW

**Review by tiffanynn — COMMENTED** [review]
😎

**Review by tiffanynn — APPROVED** [review]
😎😎

**Review by hsorathia — APPROVED** [review]

File: backend/queries/user.js [hunk 1]
```diff
@@ -19,8 +19,22 @@ exports.registerUser = (username, email, hashedPassword, admin, callback) => {
...
+        "INSERT badges_shop VALUES ('" + username + "', false, 'Get That Coin', 2500, 'Has won the coin flip at least once!');" +
+        "INSERT badges_shop VALUES ('" + username + "', false, 'Make it Rain', 5000, 'Accumulated 5000 chips in total balance. Make it rain!');" +
+        "INSERT badges_shop VALUES ('" + username + "', false, 'Blackjack Master', 10000, 'Congratulations for winning 10+ times! No need to be modest.');" +
+        "INSERT badges_shop VALUES ('" + username + "', false, 'True Winner', 10000, 'Won in Blackjack, Coin Flip, and Slots.');" +
+        "INSERT badges_shop VALUES ('" + username + "', false, '30 Day Login Streak', 15000, 'This user has logged in 30 days in a row! That is impressive.');" +
+        "INSERT badges_shop VALUES ('" + username + "', false, 'Casual Pro', 25000, 'Too pro for winning 1000 chips in total from all the games you participated in.');" +
+        "INSERT badges_shop VALUES ('" + username + "', false, 'God Chains', 50000, 'This badge is the ultimate badge to show everyone who is boss.');" +
+        "INSERT badges_shop VALUES ('" + username + "', false, 'Senpai Club', 1000000, 'The senpai badge! Senpai on the streets.');";
```
Comment (hsorathia) [inline backend/queries/user.js:36]: are these things done only once?

**Review by hsorathia — APPROVED** [review]

File: backend/queries/user.js (hunk 1 above)
Comment (realCalvin) [reply backend/queries/user.js:36]: yea. it's a long string of inserts

**Pull request merged**